packages/at_secondary_server/test/local_lookup_verb_test.dart

```diff
@@ -247,6 +250,154 @@ void main() {
     });
     tearDown(() async => await tearDownFunc());
   });
+
+  group('A group of tests related APKAM enrollment', () {
+    Response response = Response();
+    String enrollmentId = Uuid().v4();
+    setUp(() async {
+      await verbTestsSetUp();
+      inboundConnection.metadata.isAuthenticated =
+          true; // owner connection, authenticated
+    });
+
+    test(
+        'A test to verify llookup verb is allowed in all namespace when access is *:r',
+        () async {
```

Comment:
@gkc: Here is the test with *:r in the llookup verb. Any other scenarios should we add apart from this?

Reply:
Please do the same test again with `*:rw`

And add tests for 
- scan
- lookup verb, for client authenticated connection only. Note that atServer (pol) connections should be presumed always to have `*:r` / `*:rw` access for all lookups

Reply:
scan:
we have a test - A test to verify enrollment has *:rw access
@sitaram-kalluri we should add a test to check the scan results of an enrollment that has *:rw access
lookup:
authenticated lookup hits another secondary to retrieve a key. Test will be covered as part of llookup in another secondary
unauthenticated lookup can retrieve any public key. We don't restrict by access for this.
notify:
we didn't make changes for APKAM..we have to investigate and check whether any changes are required e.g. cached keys are update in notify verb handler.
@sitaram-kalluri can you please create a ticket for notify

Reply:
For now, other atServers should behave exactly as they do now - ie no namespace checking on requests from other atServers

When clients do lookup, the namespace access checks should be applied to the cache checks

Reply:
As for notify - I do believe that, for client connections, namespace access should be checked for notify:update and notify:delete commands. Otherwise, apps could spam other apps' namespaces with notifications

Reply:
@sitaram-kalluri Is there a ticket for the notify changes discussed above?